pkg/cluster/eks/defaults.go

```diff
@@ -48,81 +48,97 @@ var mappings = []struct {
 	{
 		constraintForVersion("1.11"),
 		map[string]string{
-			endpoints.UsEast2RegionID:      "ami-0f01aa60d08b56b51",
-			endpoints.UsEast1RegionID:      "ami-0277d42c17d2fd9f6",
-			endpoints.UsWest2RegionID:      "ami-0e024bc930f00f7a2",
-			endpoints.ApEast1RegionID:      "ami-01efe02a8386b4448",
-			endpoints.ApSouth1RegionID:     "ami-0fee6e6300019359b",
-			endpoints.ApNortheast1RegionID: "ami-0acea8b886f0e3f8f",
-			endpoints.ApNortheast2RegionID: "ami-01e754bb06dedfd13",
-			endpoints.ApSoutheast1RegionID: "ami-04aff8301e51a47e4",
-			endpoints.ApSoutheast2RegionID: "ami-0cb611c369549e9c9",
-			endpoints.EuCentral1RegionID:   "ami-0157324517811932f",
-			endpoints.EuWest1RegionID:      "ami-0ed6bcde59efbec8a",
-			endpoints.EuWest2RegionID:      "ami-0b2d004b35f3153c4",
-			endpoints.EuWest3RegionID:      "ami-0c592c8c15d5a2824",
-			endpoints.EuNorth1RegionID:     "ami-019c885c71264830f",
-			endpoints.MeSouth1RegionID:     "ami-0f19239ec6bfc1fd4",
+			// Kubernetes Version 1.11.10
+			"ap-east-1":      "ami-01efe02a8386b4448", // Asia Pacific (Hong Kong).
```

Comment:
What's the reason for changing to  literals for the regions instead of the constants defined in the aws-sdk?

Reply:
I think easier for generating and readable.
My script like this:
```
 K8S_VERSIONS=(
  "1.11"
  "1.12"
  "1.13"
  "1.14"
)
for version in ${K8S_VERSIONS[@]}; do
        echo "K8S Version:" $version
        for region in `aws ec2 describe-regions --output text | cut -f4 | sort -V`; do
            aws ssm get-parameter --name /aws/service/eks/optimized-ami/${version}/amazon-linux-2/recommended/image_id --region ${region} --query Parameter.Value --output text | xargs -I "{}" echo \"$region\": \"{}\",
        done
done
```